### ISSUE TITLE
then loss is inf

### ISSUE THREAD

**Issue by ll36771** [baidu-research/warp-ctc#51]
When i train the model using the warp ctc,the ctc criterion return the loss that is inf.Is there anything wrong ?How could i solve this problem?

**Comment by HulkSun** [comment]
reduce the learn_rate

**Comment by ll36771** [comment]
@HulkSun I use the adadelta criterion and i didn't set the learning rate.
